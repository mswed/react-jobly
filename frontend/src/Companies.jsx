const Companies = () => {
  return (
    <div>
      <div>Search</div>
      <div>List of Companies</div>
    </div>
  );
};

export default Companies;
