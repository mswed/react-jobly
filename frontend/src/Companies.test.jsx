import { it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Companies from './Companies';

it('renders without crashing', async () => {
  render(<Companies />);
});

it('matches snapshot', async () => {
  const { asFragment } = render(<Companies />);

  expect(asFragment()).toMatchSnapshot();
});
