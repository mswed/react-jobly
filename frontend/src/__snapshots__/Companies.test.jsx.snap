// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      Search
    </div>
    <div>
      List of Companies
    </div>
  </div>
</DocumentFragment>
`;
